front/components/assistant/AssistantDetails.tsx

```diff
@@ -287,23 +289,52 @@ export function AssistantDetails({
           {otherActions.map((action, index) =>
             isDustAppRunConfiguration(action) ? (
               <div className="flex flex-col gap-2" key={`action-${index}`}>
-                <div className="text-lg font-bold text-element-800">Action</div>
+                <div className="text-lg font-bold text-element-800">
+                  Run Actions
+                </div>
                 <DustAppSection dustApp={action} owner={owner} />
               </div>
             ) : isTablesQueryConfiguration(action) ? (
               <div className="flex flex-col gap-2" key={`action-${index}`}>
-                <div className="text-lg font-bold text-element-800">Tables</div>
-                <TablesQuerySection tablesQueryConfig={action} />
+                <div className="text-lg font-bold text-element-800">
+                  Query Tables
```

Comment:
Not a big fan of "Query Tables" as a section header.

Reply:
"Analyse data" ?

(i tried to replicate the "action description" of "retrieve ..." which i think is more appropriate ?

Reply:
Yeah I got this just it feels a bit to "rough"

Reply:
Actually, I'll keep it if you are okay because it matches the Tool name.